packages/react-components/src/components/MessageThread.tsx

```diff
@@ -712,7 +712,7 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
     }
 
     const atBottom =
-      Math.floor(chatScrollDivRef.current.scrollTop) >=
+      Math.ceil(chatScrollDivRef.current.scrollTop) >=
```

Comment:
I'm bad at math. I'd prefer to check that absolute value of difference within some tolerance.

Reply:
I was considering tolerance but didn't want to cause any unintended side effects. I'm going to leave as is to keep the delta (so hopefully any regressions) super small but I will approve anyone who wants to update this to a tolerance calculation.